core/server/data/migrations/versions/4.1/01-add-unique-constraint-for-member-stripe-tables.js

```diff
@@ -0,0 +1,14 @@
+const logging = require('../../../../../shared/logging');
+const {createIrreversibleMigration} = require('../../utils');
+const {addUnique} = require('../../../schema/commands');
+
+module.exports = createIrreversibleMigration(async (knex) => {
```

Comment:
We don't usually create irreversible migrations outside of a major, I think this should just have a `noop` for the down migration :thinking:

Reply:
@allouis Yeah good point, have changed it to `transactionalMigration` now with noop for down migration